frontend/public/style/_overrides.scss

```diff
@@ -80,6 +80,19 @@ kbd {
 
 // PatternFly 4 overrides
 
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  // reset headings font family back to correct value
+  // bug caused by PF4 _shield-inheritable.scss
```

Comment:
We should open an upstream issue if it's an upstream bug

Reply:
https://github.com/patternfly/patternfly-next/issues/2276